app/controllers/browse_everything/authorizations_controller.rb

```diff
@@ -2,6 +2,10 @@
 
 module BrowseEverything
   class AuthorizationsController < ActionController::Base
+    include BrowseEverything::Controller::JsonApiRequestable
+
+    skip_before_action :verify_authenticity_token if respond_to?(:verify_authenticity_token)
```

Comment:
These `respond_to?(:verify_authenticity_token)` were needed in order to support Rails 5.1.z releases.